client/src/components/BlipDetailSheetComponent.jsx

```diff
@@ -190,11 +194,93 @@ class BlipDetailSheetComponent extends React.Component {
         return dropdown;
     }
 
+    getCount(value) {
+        const comments = this.state.comments.filter(comment => {
+            return comment.technologie === this.props.name &&
+                comment.radar === this.props.radar
+        })
+        console.log("COMMENTS", comments)
+        var commentsOneCommentPerUser = [];
+        comments.forEach(comment => {
+             if((commentsOneCommentPerUser.findIndex(element => element.autor === comment.autor))===-1){
+                 var commentsWithSameName = comments.filter(item => {
+                     return item.autor === comment.autor;
+                 })
+                 commentsWithSameName.sort((a, b) => {
+                     return new Date(b.zeit) - new Date(a.zeit);
+                 })
+                 commentsOneCommentPerUser.push(commentsWithSameName[0]);
+             }
+        })
+        console.log("RESULT", commentsOneCommentPerUser)
+        var countValue = commentsOneCommentPerUser.filter(comment => {
+            return comment.meinung === this.state.meinungArr[value - 1];
+        }).length > 0 ? commentsOneCommentPerUser.filter(comment => {
+            return comment.meinung === this.state.meinungArr[value - 1];
+        }).length : 0;
+        console.log("Countvalue", countValue)
+        return countValue;
+    }
+
+
+    getTotalCount() {
+        return this.state.comments.filter(comment => {
+            return comment.technologie === this.props.name &&
+                comment.radar === this.props.radar
+        }).length;
+    }
+
+    getStyle(value) {
+        var countValue = this.getCount(value);
+        var count = this.getTotalCount();
+        var width = (countValue * 100 / count) + '%';
+        var style = {
+            width: width
+        };
+        return style;
+    }
+
+    getBalken() {
+        let balken = <div className="balken"></div>;
+        if (this.props.ring === "einsetzen" && (!(this.getCount(4) === 0 && this.getCount(1) === 0 && this.getCount(2) === 0)) && this.getTotalCount() >= 10) {
```

Comment:
`&& this.getTotalCount() >= 10)` muss noch raus weil wir ja gesagt haben, dass der balken doch immer sichtbar sein soll